aiven/client/cli.py

```diff
@@ -5626,6 +5631,83 @@ def organization__user__invite(self) -> None:
         """Invite user to join an organization"""
         self.client.invite_organization_user(self.args.organization_id, self.args.email)
 
+    @arg.json
+    @arg.organization_id
+    def organization__group__list(self) -> None:
+        """List user groups in an organization"""
+        groups = self.client.list_user_groups(self.args.organization_id)
+        layout = [
+            "user_group_name",
+            "description",
+            "user_group_id",
+            "member_count",
+        ]
+        self.print_response(groups, json=self.args.json, table_layout=layout)
+
+    @arg.json
+    @arg.organization_id
+    @arg.group_id_positional
```

Comment:
looks like I was overzealous and actually broke things with using `-` in the `*_positional` arg definitions, could you revert those to `_` 🙏 ?